x-pack/plugins/ml/index.js

```diff
@@ -78,6 +80,16 @@ export const ml = (kibana) => {
         // Register a function that is called whenever the xpack info changes,
         // to re-compute the license check results for this plugin
         xpackMainPlugin.info.feature(thisPlugin.id).registerLicenseCheckResultsGenerator(checkLicense);
+
+        const isEnabled = xpackMainPlugin.info.feature(thisPlugin.id).isEnabled();
+        if (isEnabled === true) {
+          addLinksToSampleDatasets(server);
+        }
+
+        const config = server.config();
+        return {
+          mlEnabled: config.get('xpack.ml.enabled'),
```

Comment:
I don't think this is needed. it's not like the `injectDefaultVars` where we're passing settings to the client.

Reply:
This was left in by mistake. Will remove it.